pkg/beholder/client.go

```diff
@@ -210,10 +210,18 @@ func NewGRPCClient(cfg Config, otlploggrpcNew otlploggrpcFactory) (*Client, erro
 	// if chip ingress is enabled, create dual source emitter that sends to both otel collector and chip ingress
 	// eventually we will remove the dual source emitter and just use chip ingress
 	if cfg.ChipIngressEmitterEnabled {
+		chipIngressOpts := []chipingress.Opt{
+			chipingress.WithTransportCredentials(insecure.NewCredentials()),
+		}
```

Comment:
See here https://github.com/smartcontractkit/chainlink-common/pull/1130/files#diff-74ce7f3d5474f67956a5d9ba5779b478b715235a0d33e8826eb99b146664ae4dR86-R92

```suggestion
		chipIngressOpts := []chipingress.Opt{
			chipingress.WithTransportCredentials(creds),
		}
```